### PR TITLE
test: add e2e tests for outputs

### LINKED ISSUE #1541: [Transactions] E2E tests: Outputs

### PR DESCRIPTION
**Pull request by jorgeepc**
This PR adds e2e tests for outputs

## Changes

- e2e tests for outputs

## Fixes

- fixes #1541 

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [x] added a test

### CODE REVIEW

File: web/cypress/e2e/TestRunDetail/Outputs.spec.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    // Save output
+    cy.wait('@getSelect');
+    cy.get('[data-cy=output-save-button]').click();
+    cy.get('[data-cy=output-pending-tag]').should('have.length', 1);
+
+    // Add new output from scratch
+    cy.get('[data-cy=output-add-button]').click();
+    cy.get('form#testOutput').within(() => {
+      cy.get('#testOutput_name').type('status_code');
+      cy.get('[data-cy=selector-editor] [contenteditable=true]').type('span[tracetest.span.type = "http"]:first');
```
Comment (xoscar) [inline web/cypress/e2e/TestRunDetail/Outputs.spec.ts:23]: I like this, what does `within` does?

File: web/cypress/e2e/TestRunDetail/Outputs.spec.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    cy.wait('@getSelect');
+    cy.get('[data-cy=output-save-button]').click();
+    cy.get('[data-cy=output-pending-tag]').should('have.length', 2);
+
+    // Publish and run
+    cy.get('[data-cy=output-publish-button]').click();
+    cy.wait('@testRuns', {timeout: 30000});
+    cy.get('[data-cy=output-item-container]').should('have.length', 2);
+
+    cy.deleteTest(true);
```
Comment (xoscar) [inline web/cypress/e2e/TestRunDetail/Outputs.spec.ts:35]: should we move create and delete tests to a before/after each cycle?

File: web/cypress/support/commands.ts [hunk 3]
```diff
@@ -53,8 +53,8 @@ Cypress.Commands.add('openTestCreationModal', () => {
 });
 
 Cypress.Commands.add('interceptTracePageApiCalls', () => {
-  cy.intercept({method: 'GET', url: '/api/tests/**/run/**'}).as('testRun');
-  cy.intercept({method: 'GET', url: '/api/tests/**'}).as('testObject');
+  cy.intercept({method: 'GET', url: /\/api\/tests\/([\w-]+)\/run\/(\w+)$/}).as('testRun');
+  cy.intercept({method: 'GET', url: /\/api\/tests\/([\w-]+)$/}).as('testObject');
```
Comment (xoscar) [inline web/cypress/support/commands.ts:57]: I like that this is more specific now

**Review by xoscar — APPROVED** [review]
GJ @jorgeepc

File: web/cypress/e2e/TestRunDetail/Outputs.spec.ts (hunk 1 above)
Comment (jorgeepc) [reply web/cypress/e2e/TestRunDetail/Outputs.spec.ts:23]: Hey, it scopes the following `cy` commands to the parent element. Very useful if you want to narrow your selectors.

**Pull request merged**